### DISCUSSION TITLE
Doris 支持 iceberg 的hadoop catalog吗？ 看官网这块没有提及

### DISCUSSION THREAD

**Discussion by Aireed** [apache/doris#22975] [A - General / Q&A] [answered]
Doris 支持 iceberg 的hadoop catalog吗？ 看官网这块没有提及

**Comment by LemonLiTree** [comment]
https://doris.apache.org/docs/dev/lakehouse/multi-catalog/iceberg/
do you need this？

**Reply by Aireed** [reply]
Thank you for your response. I have reviewed the page and found that it only supports Iceberg tables with metadata stored in Hadoop Metastore (HMS). Iceberg also has another catalog type called Hadoop, which does not require connecting to HMS, so I am not sure if it can be supported.
![image](https://github.com/apache/doris/assets/8862395/c0615a43-d16b-453f-8b37-d4e163ef66d0)

**Reply by LemonLiTree** [reply, marked as answer]
will support in the future
